src/main/java/com/vestmark/bitbucket/plugin/MergeConflictDetectorServlet.java

```diff
@@ -93,7 +93,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
     BranchClassifier bc = modelService.getModel(mcd.getToRepo()).getClassifier();
     BranchType toBranchType = bc.getType(mcd.getToBranch());
     // If the target is not master and is a release branch, find target branch and upstream releases (if any).
-    if (toBranchType.getId().equals("RELEASE") && !mcd.getToBranchId().equals("refs/heads/master")) {
+    if (!mcd.getToBranchId().equals("refs/heads/master") && toBranchType.getId().equals("RELEASE") ) {
```

Comment:
How does this prevent the 500?
is `toBranchType` null?

Reply:
Yes, toBranchType is coming back null for the master branch.  My PR header says toBranchType.getId() is null, but it is actually toBranchType that is null.